src/core/icons/fileIcons.ts

```diff
@@ -3280,5 +3280,9 @@ export const fileIcons: FileIcons = {
       name: 'metro',
       fileNames: ['metro.config.js', 'metro.config.json'],
     },
+    {
+      name: 'toc',
+      fileNames: ['toc'],
```

Comment:
@kgruiz do you mean "fileNames" or "fileExtensions" here?

The difference is that fileNames are looking for an exact match of "toc" but that doesn't inlcude "myfile.toc" file names. 

If you want to go with "myfile.toc" files then this is the way to go:

```suggestion
      fileExtensions: ['toc'],
```

The icons themselves look really nice, will merge them once you reply to this comment.